docs/docs/20-quickstart/index.md

```diff
@@ -649,18 +649,18 @@ the previous section.
 
 ## Your First Promotion
 
-1. To promote `Freight` to the `test` `Stage`, select the target icon on the
-   left border of <Hlt>test</Hlt>:
+1. To promote `Freight` to the `test` `Stage`, click the truck-arrow icon next
+   to <Hlt>test</Hlt> and then select <Hlt>Promote</Hlt>:
 
     ![Kargo-Promote](img/kargo-promote-option.png)
 
-    Next, select the `Freight` from the <Hlt>Freight Timeline</Hlt> and confirm
-    the promotion by selecting <Hlt>Yes</Hlt>:
+    Next, <Hlt>Select</Hlt> the `Freight` from the <Hlt>Freight Timeline</Hlt> and confirm
+    the promotion by selecting <Hlt>Promote</Hlt>:
 
     ![Kargo-Promote](img/kargo-promote-option-2.png)
 
-    When promotion process is complete, you'll see a check mark next to
-    <Hlt>test</Hlt>, indicating that the promotion was successful.
+    Once the promotion process completes successfully, the <Hlt>status</Hlt> field in
+    the <Hlt>Promotion</Hlt> tab of the `Stage` view shows <Hlt>Succeeded</Hlt>:
```

Comment:
This text does not match what is depicted in the image.

Reply:
> the Promotion tab of the Stage view

This is _not_ what is depicted in that image. This does indeed tell you that the Promotion has succeeded, but this is a view of the Promotion itself. It is not a view of the Stage.

The view of the Stage looks different:

![Screenshot 2025-05-21 at 11 45 04 AM](https://github.com/user-attachments/assets/c158f5d1-482b-42e7-8a65-b7c3a9e0ece9)

Also, you've not instructed the reader on how to navigate to the screen above -- not that I'm not suggesting you should _at this step_. The information they need to see is available in the Promotion view they are already looking at.

Just correct the text to more accurately describe what the reader is looking at.